eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -286,13 +286,7 @@ if (!(Test-Path $CspellConfigPath)) {
 # Lists names of files that were in some way changed between the
 # current $SourceBranch and $TargetBranch. Excludes files that were deleted to
 # prevent errors in Resolve-Path
-Write-Host "git diff --diff-filter=d --name-only --relative $TargetBranch $SourceBranch"
-$changedFilesList = git diff `
-  --diff-filter=d `
-  --name-only `
-  --relative `
```

Comment:
Do we need to handle the `--relative` parameter here?

Reply:
@danieljurek Could you verify the necessities of `--relative`?
My understanding is we git diff the relative path to current folder. The place called the script is using the working directory which is no different than no `--relative` option. Furthermore, it switched to the full path later in the call. That's why I think it is optional for current usage.

Reply:
[Posting this again in case I didn't post it somewhere else before]

I think this is OK given that `check-spelling-in-changed-files.ps1` is meant to be run from the root of the repo and doesn't change directory.